src/AsyncJson.h

```diff
@@ -35,8 +35,15 @@
 #ifndef ASYNC_JSON_H_
 #define ASYNC_JSON_H_
 #include <ArduinoJson.h>
+#include <Print.h>
 
-constexpr char* JSON_MIMETYPE = "application/json";
+#if ARDUINOJSON_VERSION_MAJOR == 5
+  #define ARDUINOJSON_5_COMPATIBILITY
+#else
+  #define DYNAMYC_JSON_DOCUMENT_SIZE  1024
```

Comment:
perhaps fix the typo to : DYNAMIC_JSON_DOCUMENT_SIZE

Reply:
I agree